apps/web/src/components/Basenames/UsernameProfileTransferOwnershipModal/AllOwnershipTransactionsState.tsx

```diff
@@ -0,0 +1,16 @@
+import { useProfileTransferOwnership } from 'apps/web/src/components/Basenames/UsernameProfileTransferOwnershipModal/context';
+import { OwnershipTransactionState } from 'apps/web/src/components/Basenames/UsernameProfileTransferOwnershipModal/OwnershipTransactionState';
+
+// List all 4 transactions states
+export default function AllOwnershipTransactionsState() {
+  const { ownershipSettings } = useProfileTransferOwnership();
+  return (
+    <ul className="flex w-full flex-col gap-4">
+      {ownershipSettings.map((ownershipSetting) => (
+        <li key={ownershipSetting.id} className="flex items-baseline gap-4">
+          <OwnershipTransactionState ownershipSetting={ownershipSetting} />
+        </li>
+      ))}
+    </ul>
+  );
+}
```

Comment:
the logic this is wrapping feels pretty simple to me, and I'd probably have just run this logic inline. It's only used in one place, and hiding the implementation inside a component makes it more challenging to parse what's going on for future readers.